sauce/features/budget/resize-inspector/index.js

```diff
@@ -0,0 +1,219 @@
+                     '<div id=toolkitInspectorIDiv class="modal" style="left: ' + btnLeft + 'px; top: ' + btnTop + 'px;">' +
+                      '<ul class="modal-list">' +
+                        '<li><button id="toolkitInspector00Btn" class="button-list"><i id=toolkitInspector00Img class="ember-view flaticon stroke checkmark-1"/>Default</button></li>' +
+                        '<li><button id="toolkitInspector25Btn" class="button-list"><i id=toolkitInspector25Img class="ember-view flaticon stroke checkmark-1"/>25%</button></li>' +
+                        '<li><button id="toolkitInspector20Btn" class="button-list"><i id=toolkitInspector20Img class="ember-view flaticon stroke checkmark-1"/>20%</button></li>' +
+                        '<li><button id="toolkitInspector15Btn" class="button-list"><i id=toolkitInspector15Img class="ember-view flaticon stroke checkmark-1"/>15%</button></li>' +
+                      '</ul>' +
+                      '<div class="modal-arrow" style="position:absolute;width: 0;height: 0;bottom: 100%;left: 37px;border: solid transparent;border-color: transparent;border-width: 15px;border-bottom-color: #fff"></div>' +
+                     '</div>' +
+                    '</div>' +
+                   '</div>');
+
+    let $00img = $modal.find('#toolkitInspector00Img');
+    let $00btn = $modal.find('#toolkitInspector00Btn');
+    $00btn.click(() => {
+      _this.setProperties(0);
+    });
+
+    let $25img = $modal.find('#toolkitInspector25Img');
+    let $25btn = $modal.find('#toolkitInspector25Btn');
+    $25btn.click(() => {
+      _this.setProperties(1);
+    });
+
+    let $20img = $modal.find('#toolkitInspector20Img');
+    let $20btn = $modal.find('#toolkitInspector20Btn');
+    $20btn.click(() => {
+      _this.setProperties(2);
+    });
+
+    let $15img = $modal.find('#toolkitInspector15Img');
+    let $15btn = $modal.find('#toolkitInspector15Btn');
+    $15btn.click(() => {
+      _this.setProperties(3);
+    });
+
+    // Handle dismissal of modal via the ESC key
+    $(document).one('keydown', (e) => {
+      if (e.keyCode === 27) { // ESC key?
+        $(document).off('click.toolkitResizeInspector');
+        $('#toolkitInspectorODiv').remove();
+      }
+    });
+
+    // Handle mouse clicks outside the drop-down modal. Namespace the
+    // click event so we can remove our specific instance.
+    $(document).on('click.toolkitResizeInspector', (e) => {
+      if (e.target.id === 'toolkitInspectorModal') {
+        $(document).off('click.toolkitResizeInspector');
+        $('#toolkitInspectorODiv').remove();
+      }
+    });
+
+    let width = toolkitHelper.getToolkitStorageKey('inspector-width', 'number');
+
+    // Determine which menu item is disabled (current width) and which
+    // items are enabled. The current width will display a checkmark
+    // while the other menu items will display nothing.
+    switch (width) { // current inspector width!
```

Comment:
That change above should allow all of this to be collapsed into about 4 lines of code.